sql/memo/join_order_builder.go

```diff
@@ -127,14 +127,15 @@ type joinOrderBuilder struct {
 	// ((xy, ab), uv), (ab, (xy, uv)), etc.
 	//
 	// The group for a single base relation is the base relation itself.
-	m             *Memo
-	plans         map[vertexSet]*ExprGroup
-	edges         []edge
-	vertices      []RelExpr
-	vertexGroups  []GroupId
-	innerEdges    edgeSet
-	nonInnerEdges edgeSet
-	newPlanCb     func(j *joinOrderBuilder, rel RelExpr)
+	m                       *Memo
+	plans                   map[vertexSet]*ExprGroup
+	edges                   []edge
+	vertices                []RelExpr
+	vertexGroups            []GroupId
+	innerEdges              edgeSet
+	nonInnerEdges           edgeSet
+	newPlanCb               func(j *joinOrderBuilder, rel RelExpr)
+	forceFastReorderForTest bool
```

Comment:
I wanted to make clear that this field is only used in tests and shouldn't be set in production code. A field name is harder to ignore than a comment.

Reply:
Renamed to forceFastDFSLookupForTest. How's that?

Reply:
Just feels weird marking an attribute as a test accessor when probably one of two things are accurate. 1) If you really want to make sure no one in GMS can use the flag, you could have tests manually call the fast path. 2) It's a legitimate optimization that we might make available via a join hint or otherwise, in which case the "test" part doesn't really add anything. In-between seems messier from an organizational standpoint. Obv not a critical thing, just an organization thing.